### DISCUSSION TITLE
Will it work with Edge browser and still being undetected?

### DISCUSSION THREAD

**Discussion by MacMarde** [kaliiiiiiiiii/Selenium-Driverless#210] [Q&A]
Will it work with Edge browser and still being undetected?

**Comment by ttlns** [comment]
I suppose so. Why not just try it yourself?

**Reply by MacMarde** [reply]
I will thanks

**Reply by ttlns** [reply]
feel free to give a feedback on that

**Comment by MacMarde** [comment]
@kaliiiiiiiiii 

I tried it with Edge but I can not make it work.

This is my code:

```
from selenium_driverless.sync import webdriver

options = webdriver.ChromeOptions()
edgePath = "C:/Program Files (x86)/Microsoft/Edge/Application/msedge.exe"
options.binary_location = edgePath
with webdriver.Chrome(options=options) as driver:
    driver.get('http://nowsecure.nl#relax')
    driver.sleep(0.5)
    driver.wait_for_cdp("Page.domContentEventFired", timeout=15)

    title = driver.title
    url = driver.current_url
    source = driver.page_source
    print(title)
```
Basically it seems to work. It starts Edge and `driver.get('http://nowsecure.nl#relax')` is also working, but then it stops here: `driver.wait_for_cdp("Page.domContentEventFired", timeout=15)`
and closes Edge with an exception (which I can not read, cause I do not know how to change my code)
I also tried it with :

```
from selenium.webdriver.edge.options import Options
options = Options()
```
But this is not working at all.

**Reply by ttlns** [reply]
just remove the line which breaks your code then

**Reply by MacMarde** [reply]
Yes you might be right, but I am worried if I might need this function.
So for what do I need it? Is it critical?

**Reply by ttlns** [reply]
nope, it's just waiting for a reload. Which probably already happened after those 0.5 seconds after load
